attestation-service/attestation-service/src/lib.rs

```diff
@@ -174,13 +172,15 @@ impl AttestationService {
             .map_err(|e| anyhow!("Verifier evaluate failed: {e:?}"))?;
 
         let flattened_claims = flatten_claims(tee, &claims_from_tee_evidence)?;
+        debug!("flattened_claims: {:#?}", flattened_claims);
```

Comment:
debug leftover

Reply:
I imported this rom your PR. Do you want to keep this? I think it's ussful for some debug cases so leave it as-is.

Reply:
sure, we can leave it